### DISCUSSION TITLE
url changed after restore

### DISCUSSION THREAD

**Discussion by FrankRoemer** [OpenCCU/OpenCCU#1899] [🇩🇪 Fragen&Antworten]
Hello,

I quite new to RaspberryMatic.

I bought an HM-MOD-RPI-PCB, put it on top of a Raspi3 and after some hickups everything worked fine.
I especially like the usage of the raspi led's. Thank you for your creative and huge work on this distribution.

I connected a door contact to proof the functionality. That worked and I tried to restore a backup from my CCU2.

That worked, but actually the url changed. That was unexpected and I elaborated a bit around that.
I guess this to be the expected behaviour, but unexpected to me. It wasn't mentioned on 
https://github.com/jens-maus/RaspberryMatic/wiki/Administration#backup--restore

Is there some "advanced documentation" e.g. how to ssh the rasbi now? How to configure the log to be written to the USBStick I put in? There are quite a few promising features mentioned but I can't find the doc to set this up.

Thank you very much

Frank

**Comment by blackbasket** [comment]
Hi Frank,

what do you mean with "the url changed"? I don't understand the issue/behavior.

Regards,
Marcel

**Comment by FrankRoemer** [comment]
Thanks for you quick reaction. 

The former url https://homematic-raspi/ was not reachable any more. I had to switch to the internal IP, like http://192.168.178.40.

Thanks
Frank

**Reply by blackbasket** [reply]
Ok, that should be the hostname of the device... I've no device accessible for now, but I think you have to take a look in the "control panel" and there maybe something like "network settings"... I think it's the same area where you can change the IP address... the device will register in DNS with this hostname... you might try to use the old CCU2's host name to validate...

**Reply by FrankRoemer** [reply]
That's true. What I was surprised, why the restore should change the hostname of the device.. especially here, when the CCU2 hostname sure was different stored than the raspi hostname.
I assumed wrongly that the rapsi nostname should not be changed by a restore of the data. Or - at least it should be mentioned in the doc that this side effect may occur. ..

**Reply by blackbasket** [reply]
It's not mentioned, because that's suspected behavior ;)

Edit: the ip settings are restored as well and anything else would be wrong
